utility/src/main/java/org/oppia/android/util/accessibility/FakeAccessibilityChecker.kt

```diff
@@ -8,12 +8,15 @@ import javax.inject.Singleton
 @Singleton
 class FakeAccessibilityChecker @Inject constructor() : AccessibilityChecker {
   private var isScreenReaderEnabled = false
-  private var announcement: CharSequence? = null
+  private var announcement: CharSequence = ""
 
   override fun isScreenReaderEnabled(): Boolean = isScreenReaderEnabled
 
-  /** Returns announcement. */
-  fun getAnnouncement(): CharSequence? = announcement
+  /**
+   * Returns latest announcement. Announcement gets overwritten each time
+   * announceForAccessibilityForView is called.
+   */
+  fun getLatestAnnouncement(): CharSequence = announcement
```

Comment:
This might actually be useful to keep as null since then it demonstrates an initial state.

Reply:
Ok, back to null